examples/CMakeLists.txt

```diff
@@ -8,9 +8,13 @@ set(IGN_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
 find_package(ignition-common2 QUIET)
 set(IGN_COMMON_VER ${ignition-common2_VERSION_MAJOR})
 if(NOT ignition-common2_FOUND)
-  find_package(ignition-common3 QUIET REQUIRED)
+  find_package(ignition-common3 QUIET)
   set(IGN_COMMON_VER ${ignition-common3_VERSION_MAJOR})
 endif()
+if(NOT ignition-common3_FOUND)
```

Comment:
I think we need an `elseif`, if it finds `common2` it doesn't need to look for `common4`